File: Javascript/2801-2850/2824. Count Pairs Whose Sum is Less than Target.js
// https://leetcode.com/problems/count-pairs-whose-sum-is-less-than-target/

var countPairs = function(nums, target) {
    let pairs = 0;
    const size = nums.length;

    for(let first =0; first < size; first++){
        for(let second = first+1; second < size ; second++){
            if(nums[first] + nums[second] < target)
                pairs++;
        }
    }

    return pairs;
};

File: Python/2801-2850/2824. Count Pairs Whose Sum is Less than Target.py
# https://leetcode.com/problems/count-pairs-whose-sum-is-less-than-target/

class Solution:
    def countPairs(self, nums: List[int], target: int) -> int:
        pairs = 0
        size = len(nums)

        for first in range(0, size):
            for second in range(first+1, size):
                if nums[first] + nums[second] < target:
                    pairs += 1

        return pairs
